src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,88 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 public class SalaryInfo {
+
+    static final class DataRow {
```

Comment:
Move this class to the end of class SalaryInfo
```suggestion
    private static final class DataRow {
```